relayer/processor/types.go

```diff
@@ -120,6 +124,15 @@ type ConnectionMessagesCache map[string]ConnectionMessageCache
 // ConnectionMessageCache is used for caching connection handshake IBC messages for a given IBC connection.
 type ConnectionMessageCache map[ConnectionKey]provider.ConnectionInfo
 
+// ClientICQType string wrapper for query/response type.
+type ClientICQType string
```

Comment:
It sounds like `ClientICQType` has only two valid values, `query` and `response`. This could be implemented as an enum to be more concise, def not a requirement though.